### DISCUSSION TITLE
Welcome to samples Discussions!

### DISCUSSION THREAD

**Discussion by yoelhor** [azure-ad-b2c/samples#409] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eskaufel** [comment]
Hi @yoelhor 

I am primarily a consumer, building B2C solutions using custom policies for customers as a consultant. 
I hope to get some time to give something back, but as always there is a timeline ⏰

-E

**Comment by esteban-sandoval** [comment]
Hi @yoelhor 
I'm building B2C solutions to customers using custom policies as @eskaufel said. 
Related to this example, https://github.com/azure-ad-b2c/samples/blob/master/policies/groups
That is very well! I'm having a more challenge situation, where I need to "parse" or format the API Graph results as you did, with Group_1, Group_2 and Group_3 ClaimsTransformations, but the difficult for me is that the amount of groups that the API Graph return is dynamic, I mean, each user have differents groups (and amount of). 
It is possible to keep the same idea (that work perfect for 3 groups!) but with an unknown amount of returned groups? (the JSON returned). 

Another little thing that I realized is that the user needs to have exactlly 3 groups, if he has less, the B2C id_token come without "groups" claim.

I realy apreciate any thing that can help me in this challenge!!
